sample_programs/cpp_sample_programs/factorial/input.yaml

```diff
@@ -7,13 +7,9 @@ compileOption:
             - ret
 
     regSelMethod: regloc
-    regloc: dstreg
+    regloc: allreg
```

Comment:
Should we also add this option `allreg` to the master YAML file, under `docs/input_masterlist.yaml` and `docs/input_masterlist_ml.yaml` ?

Reply:
Yes, we should - I didn't know how to do this via a PR though. The easiest way would be to directly edit the Wiki page